### PR TITLE
Device branch

### PR DESCRIPTION
**Pull request by Divinity-dev**
In this phase of the project, I have implemented the following, by adding a device;
1. Allow a user to register
2. User can log in with email and password
3. password stored in a database
4. reset the password 
5. ask for confirmation of email

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,11 +1,21 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (VICTOR-CHUKWU) [inline app/models/user.rb:4]: - [ ] The application should ask for confirmation by email. kindly add confirmable to devise

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Divinity-dev  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed
- devise added


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Divinity-dev, Team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with authentication. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
Thank you 

Sent from Yahoo Mail on Android 
 
  On Thu, May 25, 2023 at 12:08 PM, Ranjeet ***@***.***> wrote:   


@thecodechaser approved this pull request.

STATUS: APPROVED 🟢
 
Hi @Divinity-dev, Team 👋 ,

Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉
 
To Highlight 🎉
    
   - Good job with authentication. ✔️
   - No linter errors. ✔️
   - Gitflow is followed. ✔️
   - Professional README file. ✔️
   - PR has a good title and summary. ✔️



Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**